cli/ops/runtime.rs

```diff
@@ -28,16 +29,13 @@ fn op_start(
   _zero_copy: Option<ZeroCopyBuf>,
 ) -> Result<JsonOp, ErrBox> {
   let gs = &state.global_state;
-  let script_args = if gs.flags.argv.len() >= 2 {
-    gs.flags.argv.clone().split_off(2)
-  } else {
-    vec![]
-  };
+
   Ok(JsonOp::Sync(json!({
     "cwd": deno_fs::normalize_path(&env::current_dir().unwrap()),
     "pid": std::process::id(),
-    "argv": script_args,
-    "mainModule": gs.main_module.as_ref().map(|x| x.to_string()),
+    "args": gs.flags.argv.clone(),
+    "repl": gs.flags.subcommand == DenoSubcommand::Repl,
```

Comment:
`repl` is very good candidate for a separate entry point - just like we did with `worker` - that should help working on it in the future.